lesson_11/test_homework_11.py

```diff
@@ -0,0 +1,33 @@
+import unittest
```

Comment:
You've done a great job. For the future - I can provide you an advice. I guess you see that you code for file opening is duplicated for all test cases. The great practice will be to create separate function that receives file as an argument, opens it and returns desired logging message as expected result.